File: src/main/java/omerozturk/hrms/business/abstracts/ResumeService.java
package omerozturk.hrms.business.abstracts;

import omerozturk.hrms.core.utilities.result.DataResult;
import omerozturk.hrms.core.utilities.result.Result;
import omerozturk.hrms.entities.concretes.*;
import omerozturk.hrms.entities.dtos.*;

import java.util.List;

public interface ResumeService {
    Result add(ResumeDto resumeDto);
    DataResult<List<Resume>> getAll();
    DataResult<Resume> getById(int resumeId);
    Result update(ResumeDto resumeDto);
    DataResult<Resume> getByEmployeeIdAndDate(int employeeId);
}

File: src/main/java/omerozturk/hrms/uiPresentation/src/services/employeeServices/resumeService.js
import axios from "axios";

export default class ResumeService {
    getAll() {
        return axios.get("http://localhost:8080/api/resumes/getall");
    }
    getById(id) {
        return axios.get("http://localhost:8080/api/resumes/getById?resumeId=" + id);
    }
    getByEmployeeIdAndDate(id) {
        return axios.get("http://localhost:8080/api/resumes/getByEmployeeIdAndDate?employeeId=" + id);
    }

    add(resume) {
        return axios.post('http://localhost:8080/api/resumes/add', resume)
            .then(function (response) { console.log(response); })
            .catch(function (error) {
                console.log(error);
            });
    }
}